frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -396,33 +398,66 @@ object Project {
       options ++ ysemanticdb ++ sourceRoot
     }
 
+    def enableJavaSemanticdbOptions(options: List[String]): List[String] = {
+      if (hasJavaSemanticDBEnabledInCompilerOptions(options)) options
+      else
+        s"-Xplugin:semanticdb -sourceroot:${workspaceDir} -targetroot:javac-classes-directory" :: options
```

Comment:
No.  Unlike the Scalac plugin options which are all prefaced with `-P:semanticdb:` and feed down into the plugin, the javac one has no preface and has to be a single option.  Otherwise javac thinks `-sourceroot` is a standalone compiler option and doesn't recognise it